--- pari-2.11.1/config/get_config_options.orig	2019-01-11 23:41:58.617375425 +0100
+++ pari-2.11.1/config/get_config_options	2019-01-11 23:43:54.143362777 +0100
@@ -87,4 +87,5 @@
       with_ncurses_lib=`echo "$1" | sed -e 's/[-a-z]*=//'` ;;
 
+  --without-qt|--with-qt=no) without_qt=yes ;;
   --with-qt) with_qt=yes ;;
   --with-qt=*)
@@ -95,4 +96,5 @@
       with_qt_include=`echo "$1" | sed -e 's/[-a-z]*=//'` ;;
 
+  --without-fltk|--with-fltk=no) without_fltk=yes ;;
   --with-fltk) with_fltk=yes ;;
   --with-fltk=*)
@@ -173,8 +175,10 @@
 
   --with-qt[=DIR]            use the Qt graphical library [prefix for Qt dir.]
+  --without-qt               do not try to use the Qt graphical library
   --with-qt-include=DIR      specify location of Qt headers
   --with-qt-lib=DIR          specify location of Qt libs
 
   --with-fltk                use the FLTK graphical library [need fltk-config]
+  --without-fltk             do not try to use the FLTK graphical library
 
 Miscellaneous
